src/main/java/com/google/ehub/data/FavoriteItemDatastore.java

```diff
@@ -19,6 +21,9 @@
   private static final String USER_EMAIL_PROPERTY_KEY = "userEmail";
   private static final String ITEM_ID_PROPERTY_KEY = "itemId";
 
+  private static final String ENTERTAINMENT_ITEM_KIND = "entertainmentItem";
+  private static final String NUMBER_OF_LIKES_PROPERTY_KEY = "numberOfLikes";
```

Comment:
Instead of creating new constants here, its better to reuse the constants from EntertainmentItemDatastore. One reason we store the hardcoded strings in constants is to be able to reuse them